apps/test-suite/tests/ImagePicker.js

```diff
@@ -55,14 +61,14 @@ export async function test({ it, xit, beforeAll, expect, jasmine, xdescribe, des
     describe('launchCameraAsync', () => {
       if (Constants.isDevice) {
         it('launches the camera', async () => {
-          alert('Please take a picture for this test to pass.');
+          await alertWithInstructions('Please take a picture for this test to pass.');
```

Comment:
Sorry for that, I've added `non-await` alert during iOS QA where it worked and haven't tested it on Android 🤦‍♂️